docs/src/man/missing.md

```diff
@@ -63,6 +63,46 @@ julia> coalesce.(x, 0)
 
 ```
 
+The function `dropmissing` or `dropmissing!` can be used to remove the rows with incomplete data from a DataFrame and either create a new DataFrame or mutate the original in-place. 
+
+```jldoctest missings
+julia> df = DataFrame(i = 1:5,
+                      x = [missing, 4, missing, 2, 1],
+                      y = [missing, missing, "c", "d", "e"])
+
+julia> dropmissing(df)
+2×3 DataFrame
+│ Row │ i     │ x      │ y       │
+│     │ Int64 │ Int64⍰ │ String⍰ │
+├─────┼───────┼────────┼─────────┤
+│ 1   │ 4     │ 2      │ d       │
+│ 2   │ 5     │ 1      │ e       │
+```
+By default dropmissing keep the `Union{Type,Missing}` type. To remove the Missing part, set the `disallowmissing` option to true (it should become the default in the future).

```

Comment:
```suggestion
By default the `dropmissing` and `dropmissing!` functions keep the `Union{Type,Missing}` element type in columns selected for row removal. To remove the `Missing` part, if present, set the `disallowmissing` option to `true` (it will become the default behavior in the future).
```